errors/runconfig.go

```diff
@@ -0,0 +1,204 @@
+package errors
+
+// This file contains all of the errors that can be generated from the
+// docker/daemon component.
+
+import (
+	"net/http"
+
+	"github.com/docker/distribution/registry/api/errcode"
+)
+
+var (
+	// ErrorCodeInvalidMacAddress is generated when invalid Mac Address is provided.
+	ErrorCodeInvalidMacAddress = errcode.Register(errGroup, errcode.ErrorDescriptor{
+		Value:          "INVALIDMACADDRESS",
```

Comment:
The patterns is clearly shown in registry/api/v2/errors.go in the distribution project. They should be namespaced and have underscores.